pkg/pipeline/transform/kubernetes/kubernetes_test.go

```diff
@@ -47,50 +47,161 @@ func (indexMock *IndexerMock) ByIndex(indexName, indexedValue string) ([]interfa
 	return args.Get(0).([]interface{}), args.Error(1)
 }
 
+func (indexMock *IndexerMock) GetByKey(key string) (interface{}, bool, error) {
+	args := indexMock.Called(key)
+	return args.Get(0), args.Bool(1), args.Error(2)
+}
+
 func (informerMock *InformerMock) GetIndexer() cache.Indexer {
 	args := informerMock.Called()
 	return args.Get(0).(cache.Indexer)
 }
 
-func TestGetInfoPods(t *testing.T) {
-	kubeData := KubeData{}
-	// pods informer
-	pidx := IndexerMock{}
-	pidx.On("ByIndex", IndexIP, "1.2.3.4").Return([]interface{}{&Info{
+func (m *IndexerMock) mockPod(ip, name, namespace, nodeIP string, owner *Owner) {
```

Comment:
There is inconsistency in the receiver variable name of `*IndexerMock`. Sometimes it's `indexMock` and sometimes it's `m`. I think it'll make sense to rename the instances of `indexMock` to `m` (or vice versa).